dockerfile

```diff
@@ -9,4 +9,4 @@ RUN pip3 install --upgrade python-owasp-zap-v2.4
 
 WORKDIR /zap
 
-copy wavsep wivet /zap/
+copy wavsep wivet vulnerableApp /zap/
```

Comment:
this is the change which is i think needed for the scope api to run @psiinon

Reply:
@psiinon i am getting following error while running this action: https://github.com/preetkaran20/zap-mgmt-scripts/runs/1054469186?check_suite_focus=true

can you please help with this.

Reply:
I think we need to first build the docker owasp/testing image and then we can try. How were you able to handle this.

Reply:
As per my previous comment :) I built it locally and pushed it to psiinon/zap-testing - I also changed my copy of the action to use that instead.